configure.ac

```diff
@@ -924,6 +938,7 @@ echo "Building Python Library ...... : $have_python"
 echo "Building Py3 Library ......... : $have_py3"
 echo "Building Ruby Library ........ : $have_ruby"
 echo "Building Rust Library ........ : $have_rs"
+echo "Preparing Common Lisp tests... : $have_cl"
```

Comment:
why isn't it `Building Common Lisp Library`?

Reply:
I think that would be confusing. The thing is, when you hit `make`, the Common Lisp library will not be built at all, unlike the others. It's only baked into the cross-test binaries when you do `make precross` or `make cross`.